### PR TITLE
Improve improved nested inputs release notes section

### PR DESCRIPTION
**Pull request by wolfs**

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -66,24 +66,35 @@ The IDEA plugin now automatically marks your Java resource directories (e.g. `sr
...
-### Support for mapped and named nested inputs
+### Support for mapped nested inputs
 
 When dealing with task inputs, it may be that not all values are known upfront.
-For example, for the `findbugs` plugin, it is not clear at configuration time which reports will be enabled by the time the task executes.
-For each report itself it is easy to declare the inputs and outputs, this is just a matter of annotating the concrete report class.
-Now it is also easy to [declare the map of enabled reports](https://github.com/gradle/gradle/blob/2376cd3824ea683c1af122f8a582ceb6ef51ec3b/subprojects/reporting/src/main/java/org/gradle/api/reporting/internal/DefaultReportContainer.java#L121-L124) as an input:
+For example, for code quality plugins like e.g. the [`findbugs`](userguide/findbugs_plugin.html), it is possible to configure the creation of different kinds of reports, e.g. generating an HTML report and a plain text report.
+
+For each report itself it is easy to declare the inputs and outputs, this is just a matter of annotating the concrete class used to configure the report.
```
[the remark this answers is not in the document]
Comment (wolfs) [reply subprojects/docs/src/docs/release/notes.md]: I added a before section. The DSL configuration is the same in both cases.

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -66,24 +66,35 @@ The IDEA plugin now automatically marks your Java resource directories (e.g. `sr
...
...M9Us/JySUvrz5IsvM7c5EIr8I3116U239kJIHNMD2AKJK/6RcsW3gEqV2xka49Bfy9vx/nSmufmKlKgIQiVYmvAgFF6FAj8hF0AY8RGQCT2npYt9eEsriE4qbl+R2mDEhIMqxW4aU/FxWFtAKJ6jxsoFrgkSB5h4Ty0PalrYhhKiwLGIRzZEgIb2IQygxBIJAwShyOqvPyVOqE0l2sRNcVc+oOEhjw0IhQCLGYlxjr2u6M3H9YX0ciCU1heRRsypUeZUY+CN5OtO6miVOTHRaUqAUOiSzgcV6QbR+AxqjSA9/mx/n2NOb9N2z71FzUR5XEKrXQESZqEdhGq3/MNb+wkQShtcDPvpsIAkciAU0o5gq8YSZwpCIWN9JWxmOnpUPE9jI3qojdYjQEmhXUjGiYF5EKrZauWpunYqeh6rZSUaC/AihQQ1jjOoI5/I4ceqLrPslKvxD0IhsRwIlTBZeQpnqKXwjKv9/T9+bgQKmQahBIFB+NjYRhZjHoQaMzPSgVAddSAURKEiARwqAR5BqO0OOVJ1r2OfOn2m9uddfX0OhJpu+dV1Kq1xIFQW58QZb9yDsZW320XFqlzZinfn8Sfn+/icA6FIyNJXfpse77njz/o/M2CitDkyoYIVyWJgfGpG7fYWEOr/l1iBrxbZaIsIQiVYmvAgFCtLGE1RaQUvsFLI3DEGR8ePD8QGU3f77ID68SCar82Jra7fy3S0sbWAUOymBUt2pzzA9kiM6AWPH8Z4DC74xg4liRiw3ZKe3S4KXBA+18nxME5DlD87/QBBKOAbR0eRMmoTHOt6rEiFZj0eHzMQSqqFJPudufCj6bo3bwiPJJocCKWQ3suzCMAiCCX9QXREI4yCu+M1NoJQI1TmMNGjB6EIHQnkkDNKy/QVIJQL1TUGDIk1wqS3aI4op7dpuydtUXIEN0WjnMnC6kOLdGaqXQHazUIahGq0/ENa+wkQStVSWL8eskNBJOallwah6KH+8CaSFUChOGBH+t3qEAzIOjE1PhbHFfJBnIzDMpR7gK1TyzwI1Wy18tSwdup6biSOKUs01p918l7QrrrKUIOwmWkQCnOKkteJweZAqDi/CRAqYbIGRUyK1oKWFdCSkpcjsRZQcjSWNQgmy5KR3UiDUMYcXcme3Y11lwahxsiMdCBURx0IxWqfe60NVaL7mjvuiiAUJcxVfZzPkV59
```
[the remark this answers is not in the document]
Comment (wolfs) [reply subprojects/docs/src/docs/release/notes.md]: Added some highlighting.

**Comment by pioterj** [comment]
Looks much better, thanks. Is it also documented in the user guide?

**Comment by pioterj** [comment]
One thing that is still a bit unclear for me is the term "nested input". Can't nested type contain output as well? I this case the report destination is actually output, right?

**Pull request merged**